docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+# Configuring z/OSMF Security
+
+Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+
+:::info **Required role: security administrator**
+:::
+
+## Configuring z/OS REST services SAF security
+
+A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe CLI requires. Zowe CLI **[correct?]** uses REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that Zowe CLI **[correct?]** can communicate with z/OS systems.
+
+:::caution
+
+Before you allow users to issue z/OS console commands with Zowe CLI **[correct?]**, security administrators should ensure that they provide access to commands that are appropriate for their organization.
+
+:::
+
+The following table details the required z/OSMF REST services and examples of the Zowe CLI **[correct?]** features they enable. If the profile validation command returns any errors, use this table to find IBM documentation for the z/OSMF REST APIs.
+
+| z/OSMF REST Service        | REST Endpoint | Description | More information |
+| ----------- | ----------- | ---------- | ------------- |
+| Cloud provisioning services | Endpoints that begin with: `/zosmf/provisioning/` | Cloud provisioning for development environments. | <ul><li>[Cloud provisioning services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-cloud-provisioning)</li></ul> |
+| TSO/E address space services | Endpoints that begin with: `/zosmf/tsoApp` | TSO commands (`zowe zos-tso issue`). | <ul><li>[TSO/E address space services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-tsoe-address-space)</li><li>[Class activations that z/OSMF requires](https://www.ibm.com/docs/en/zos/2.2.0?topic=guide-security-configuration-requirements-zosmf#DefaultSecuritySetupForZosmf__ResourceAuthorizationsForRESTapi)</li></ul> |
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for the v2.2 article "TSO/E address space services" in the IBM Docs. Could one of you help me find a replacement?

Reply:
This one I am less sure about, but I think it is this one.
https://www.ibm.com/docs/en/zos/2.5.0?topic=guide-security-structures-zosmf#DefaultSecuritySetupForZosmf__GenericClassActivationsThatRequires__title__1

Reply:
Yeah, there wasn't a section that was just TSO/E address space services, and I can't link to a list element. :(